api/schema/simple.json

```diff
@@ -1 +1,4 @@
-{}
\ No newline at end of file
+{
+  "title": "Definition",
+  "type": "object"
+}
```

Comment:
Is this required at all? If you're using this as a starting point, then this is a good place to start; but if you're happy with the existing `OPDSFeed` implementation, I don't think we need to introduce JSON Schema into this project until we have it as an easy-to-use package.

Reply:
Yeah, forgot to comment about this before, but this is required at a minimum for the builder to work - it needs a file to the schema. And the schema needs to have this as a minimum, too.

Reply:
I'm fine with this, but I think the tiny schema may be causing your test failure.